AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -280,6 +351,7 @@ void GCS_MAVLINK_Tracker::packetReceived(const mavlink_status_t &status,
 {
     // return immediately if sysid doesn't match our target sysid
     if ((tracker.g.sysid_target != 0) && (tracker.g.sysid_target != msg.sysid)) {
+        GCS_MAVLINK::packetReceived(status, msg);
```

Comment:
@peterbarker Adding this fixed the communication lock out once connected, i think it is the right thing to do?

Reply:
It will definitely do!

Doing it neater would be to break out a `handle_target_packet(status, msg)` and calling that in the case the packet IS from our tracking target.  Regardless of that you'd call `packetReceived` in only one place, after that if statement.  This is a larger change, of course, so it can be done later.